nosetimer/plugin.py

```diff
@@ -99,7 +99,7 @@ class TimerPlugin(Plugin):
     name = 'timer'
     score = 1
 
-    time_format = re.compile(r'^(?P<time>\d+)(?P<units>s|ms)?$')
+    time_format = re.compile(r'^(?P<time>\d+\.?\d*)(?P<units>s|ms)?$')
```

Comment:
Do we want to limit the number digits after the decimal point? How would something like `5.322134` work?

Reply:
Sorry, I am not following what the problem would be?  I don't expect people specifying this specific floating point numbers, but they are no worse than specific int ms.